tools/convert_votenet_checkpoints.py

```diff
@@ -0,0 +1,108 @@
+import argparse
+import torch
+from mmcv import Config
+from mmcv.runner import load_state_dict
+
+from mmdet3d.models import build_detector
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='MMDet test (and eval) a model')
+    parser.add_argument('config', help='test config file path')
```

Comment:
The checkpoint should contain config in meta, so maybe we can directly use the config in checkpoint without loading config.